ush/build.sh

```diff
@@ -30,12 +30,13 @@ set -x
 
 # Set CONTROLPATH variable to user develop installation
 CONTROLPATH="$DIR_ROOT/../develop/install/bin"
-
+CMAKELIBS=".so .a"
+CMAKE_OPTS+=" -DCMAKE_FIND_LIBRARY_SUFFIXES=${CMAKELIBS}"
```

Comment:
I don't think this is necessary.

Reply:
Removed. The code compiles and runs without these options specified explicitly